README.md

```diff
@@ -3627,22 +3627,50 @@ resource cleanup when possible.
 <sup>[[link](#percent-s)]</sup>
 
 * <a name="percent-literal-braces"></a>
-  Prefer `()` as delimiters for all `%` literals, except `%r`. Since parentheses
-  often appear inside regular expressions in many scenarios a less common
-  character like `{` might be a better choice for a delimiter, depending on the
-  regexp's content.
-<sup>[[link](#percent-literal-braces)]</sup>
+  Prefer brackets as delimiter for all `%` literals, because their usage does 
+  not need to be escaped.
 
   ```Ruby
   # bad
-  %w[one two three]
-  %q{"Test's king!", John said.}
+  %-"#{language} \-is\- awesome!"-
+  %!"#{language} (is) awesome\!"!
 
   # good
+  %("#{language} -is- awesome!")
+  %("#{language} (is) awesome!")
+  ```
+
+  Use meaningful brackets as delimiters, like `[]` for Arrays
+
+  ```Ruby
+  # bad
   %w(one two three)
-  %q("Test's king!", John said.)
+  %i<one two three>
+
+  # good
+  %w[one two three]
+  %i[one two three]
   ```
 
+  and `()` for Strings and shell commands
```

Comment:
I don't like sentences that are broken up by code examples. It makes it really hard to follow such text along.